lib/src/main/java/vmath/Vec2.java

```diff
@@ -49,6 +49,13 @@ public float dot(Vec2 other) {
         return x*other.x + y*other.y;
     }
 
+    public Vec2 lerp(Vec2 end, float t) {
```

Comment:
We should use the name ```other``` like in the issue statement. But that is just my opinion